src/app.ts

```diff
@@ -38,6 +39,16 @@ const library = builder.buildLibrary();
 
 const outputCQLFile = path.join(program.output, `${igJson.name}.cql`);
 fs.writeFileSync(outputCQLFile, library.cql, 'utf8');
+
+logger.info('bundling ValueSets');
+const valueSets = fs
+  .readdirSync(igDir)
+  .filter((f: string) => path.extname(f) === '.json' && f.startsWith('ValueSet'))
+  .map((vs: string) => JSON.parse(fs.readFileSync(path.join(igDir, vs), 'utf8')));
+
+const bundle = bundlify(valueSets);
+fs.writeFileSync(path.join(program.output, `${igJson.name}-valuesets.json`), JSON.stringify(bundle), 'utf8');
```

Comment:
Is this bundling the valuesets and then taking that bundle and putting into another bundle?

Reply:
It takes all the individual valueset resources in the directory and puts them all into a collection bundle:

```
{
  "resourceType": "Bundle",
  "type": "collection",
  "entry": [
    {
      "resource": {
        "resourceType": "ValueSet",
        "id": "mcode-cancer-body-location-vs",
        "text": {
          "status": "generated",
          "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><h2>Cancer Body Location Value Set</h2><div><p>Codes describing the location(s) of primary or secondary cancer. The value set includes all codes from the SNOMED CT body structure hierarchy (codes descending from 123037004 'Body Structure'). The cancer body location may also be expressed using ICD-O-3 topography codes, however, those codes are not included here due to intellectual property restrictions. No other code systems are considered conformant.</p>\n</div><ul><li>Include codes from <a href=\"http://www.snomed.org/\"><code>http://snomed.info/sct</code></a> where concept  is-a  123037004 (Body structure)</li></ul></div>"
        },
        "url": "http://hl7.org/fhir/us/mcode/ValueSet/mcode-cancer-body-location-vs",
        "version": "1.0.1",
        "name": "CancerBodyLocationVS",
        "title": "Cancer Body Location Value Set",
        "status": "active",
        "date": "2020-03-30T01:35:33+00:00",
        "publisher": "HL7 International Clinical Interoperability Council",
        "contact": [
          {
            "name": "HL7 International Clinical Interoperability Council",
            "telecom": [
              {
                "system": "url",
                "value": "http://www.hl7.org/Special/committees/cic"
              },
              {
                "system": "email",
                "value": "ciclist@lists.HL7.org"
              }
            ]
          }
        ],
        "description": "Codes describing the location(s) of primary or secondary cancer. The value set includes all codes from the SNOMED CT body structure hierarchy (codes descending from 123037004 'Body Structure'). The cancer body location may also be expressed using ICD-O-3 topography codes, however, those codes are not included here due to intellectual property restrictions. No other code systems are considered conformant.",
        "compose": {
          "include": [
            {
              "system": "http://snomed.info/sct",
              "filter": [
                {
                  "property": "concept",
                  "op": "is-a",
                  "value": "123037004"
                }
              ]
            }
          ]
        }
      }
    }
   ....
]
}
```